react-app/package.json

```diff
@@ -2,9 +2,24 @@
   "name": "my-app-using-romcal",
   "version": "0.1.0",
   "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "tsc && NODE_NO_WARNINGS=1 vite build",
+    "build:dev": "VITE_DAY_VARIANT=developer npm run build",
+    "dev:link": "ROMCAL_ALIAS=\"../${ROMCAL_ALIAS:-romcal}\" bash ../scripts/link.sh",
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "prepare": "npx --yes update-browserslist-db@latest",
+    "serve": "vite preview",
+    "start": "NODE_NO_WARNINGS=1 vite",
+    "start:dev": "VITE_DAY_VARIANT=developer npm start",
+    "start:built": "NODE_NO_WARNINGS=1 npx http-server build -p 3000 -g -s",
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
will do, these are related to others above, but i'll leave the convo unresolved so i don't miss it, but to summarize:
- `ROMCAL_APP_PORT` - will implement
- `serve` for dynamic, `start` for prebuilt -> will do.

I think we could probably just drop off `vite preview`, because `"serve": "NODE_NO_WARNINGS=1 vite"`would cover our usages in my opinion